pkg/controller/client/client.go

```diff
@@ -13,44 +13,52 @@ type Client struct {
 }
 
 // NewControllerClient return a new Client
-func NewControllerClient(url string, auth request.Authenticator) controller.Interface {
-	client := request.NewClient().BaseURL(url).Encoder(json.Marshal).Decoder(request.JSONDecoderGenerator).Authenticator(auth).Build()
+func NewControllerClient(
+	url string,
+	auth request.Authenticator,
+) controller.Interface {
```

Comment:
As there aren't that many parameters, I find it more readable to have these in just one line (this can be discussed with the rest of the team)